org.eclipse.scanning.api/src/org/eclipse/scanning/api/points/models/MultiStepModel.java

```diff
@@ -35,11 +45,22 @@ public MultiStepModel(String name, double start, double stop, double step) {
 		stepModels.add(new StepModel(name, start, stop, step));
 	}
 	
+	/**
+	 * Must implement clear() method on beans being used with BeanUI.
+	 */
+	public void clear() {
+		List<StepModel> oldModels = new ArrayList<StepModel>(stepModels);
+		stepModels.clear();
+		firePropertyChange("stepModels", oldModels, stepModels);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "stepModels" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)